library/traefik

```diff
@@ -20,42 +20,22 @@ GitRepo: https://github.com/traefik/traefik-library-image.git
 GitCommit: 66250e268eb9c8d592cc85e153d388dca705c7ee
 Directory: alpine
 
-Tags: v2.11.0-rc2-windowsservercore-ltsc2022, 2.11.0-rc2-windowsservercore-ltsc2022, v2.11-windowsservercore-ltsc2022, 2.11-windowsservercore-ltsc2022, mimolette-windowsservercore-ltsc2022
+Tags: v2.11.0-windowsservercore-ltsc2022, 2.11.0-windowsservercore-ltsc2022, v2.11-windowsservercore-ltsc2022, 2.11-windowsservercore-ltsc2022, mimolette-windowsservercore-ltsc2022, windowsservercore-ltsc2022
 Architectures: windows-amd64
 GitRepo: https://github.com/traefik/traefik-library-image.git
-GitCommit: 17a3fad865ef2fc8ddd0fe25dce0a470f0fe78a4
+GitCommit: fccb66b9845fc4266154377b774ac95127d38078
 Directory: windows/servercore-ltsc2022
 Constraints: windowsservercore-ltsc2022
 
-Tags: v2.11.0-rc2-windowsservercore-1809, 2.11.0-rc2-windowsservercore-1809, v2.11-windowsservercore-1809, 2.11-windowsservercore-1809, mimolette-windowsservercore-1809
+Tags: v2.11.0-windowsservercore-1809, 2.11.0-windowsservercore-1809, v2.11-windowsservercore-1809, 2.11-windowsservercore-1809, mimolette-windowsservercore-1809, windowsservercore-1809
 Architectures: windows-amd64
 GitRepo: https://github.com/traefik/traefik-library-image.git
-GitCommit: 17a3fad865ef2fc8ddd0fe25dce0a470f0fe78a4
+GitCommit: fccb66b9845fc4266154377b774ac95127d38078
 Directory: windows/1809
 Constraints: windowsservercore-1809
 
-Tags: v2.11.0-rc2, 2.11.0-rc2, v2.11, 2.11, mimolette
+Tags: v2.11.0, 2.11.0, v2.11, 2.11, mimolette, latest
 Architectures: amd64, arm32v6, arm64v8, s390x
```

Comment:
I see `+		ppc64le) arch='ppc64le' ;; \` in the Dockerfile diff, did you want us to build `ppc64le` images too?  If so, it should be added to this `Architectures:` list.

Reply:
Feel free to add the architecture in a follow up PR. Merging now to get the new release out.